### PR TITLE
Add snyk

### PR DESCRIPTION
**Pull request by guypod**
Vulnerable dependencies are a real and growing problem.

Roughly 1 in 7 npm packages carries a known vulnerability, and at Snyk we see 4 out of 5 users find vulnerabilities in their apps. Roughly 40 new vulnerabilities were disclosed in the last 6 months, many of which impacting hugely popular packages like request, mongoose, qs and more. 

Snyk gives npm users a free and easy way to find these vulnerable dependencies, upgrade them away when they can, patch them when they can't (with open source patches from https://github.com/Snyk/vulndb) and monitor to be alerted when new vulnerabilities are disclosed on their dependencies.

Full disclosure: I'm a part of the Snyk team, you can see the rest of the team at https://snyk.io/about/

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -54,13 +54,13 @@ You might also like [awesome-nodejs](https://github.com/sindresorhus/awesome-nod
...
 - [npm addict](https://npmaddict.com) - Your daily injection of npm packages.
 - [npm discover](http://www.npmdiscover.com) - Discover what packages are commonly used together.
+- [snyk](https://snyk.io) - Find, fix and monitor for known vulnerabilities in npm dependencies.
 
 ### Browser extensions
 
 - [Octo-Linker](https://chrome.google.com/webstore/detail/octo-linker/jlmafbaeoofdegohdhinkhilhclaklkp) - Chrome extension to navigate across npm packages on GitHub with ease.
 - [npm-hub](https://chrome.google.com/webstore/detail/npm-hub/kbbbjimdjbjclaebffknlabpogocablj) - Chrome extension to explore npm dependencies on GitHub repos.
 
-
```
Comment (sindresorhus) [inline readme.md:63]: Don't do unrelated changes.

Comment (guypod) [inline readme.md:63]: Line was removed by accident, PR now updated to put it back in.
